src/corehost/cli/deps_resolver.cpp

```diff
@@ -415,6 +454,109 @@ void deps_resolver_t::get_local_assemblies(const pal::string_t& dir)
 }
 
 // -----------------------------------------------------------------------------
+// Patch mscorlib from package relative dir if not sxs coreclr.
+//
+// TEMPORARY WORKAROUND for NuGet packages that don't contain sxs mscorlib.
```

Comment:
We should file an issue to remove this once the package comes with mscorlib.ni.dll